cypress/e2e/custom_data_layer/sql_alchemy.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+
+import chainlit.data as cl_data
+from chainlit.data.sql_alchemy import SQLAlchemyDataLayer
+from literalai.helper import utc_now
+
+import chainlit as cl
+
+cl_data._data_layer = SQLAlchemyDataLayer(conninfo="<your conninfo>")
```

Comment:
Shall we add blob storage client for testing?

Reply:
done!